### DISCUSSION TITLE
Acceptance of ASReview in Systematic Reviews Among Researchers

### DISCUSSION THREAD

**Discussion by soldiHo** [asreview/asreview#1695] [Q&A]
I'm considering using ASReview for my thesis's systematic review at Utrecht University. My supervisor, however, expresses concerns that utilizing ASReview might lead to identifying a large number of articles, e.g., 1000, and despite screening 200 of them and applying a cut-off score according to the SAFE procedure, there's a risk that examiners or second reviewers might insist on screening all identified articles.

To researchers who have integrated ASReview into their systematic reviews: Have you encountered feedback suggesting that all articles should be screened? How prevalent is this concern?

**Comment by Rensvandeschoot** [comment]
Great you are planning to use ASReview! The method, active learning or screening prioritization has been accepted by the PRISMA guidelines: https://github.com/asreview/asreview/discussions/979#discussioncomment-2195361. 

Your question is also a nice shootout to researchers who have experience publishing with such methods, so let's hope some will reply!!

**Reply by FelixWdm** [reply]
Hi @SoldadoK, we get this question frequently. We don't know of any examples of reviewers suggesting screening all. But we keep track of papers that cite or ASReview: https://www.zotero.org/groups/4597652/asreview_public/collections/IWU8ATMK.
